.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,26 @@
+name: Node CI
```

Comment:
The `npm t` should run all the tests. This extra action config seems to be not necessary.